scripts/manual_operations/tests/test_manual_remove.py

```diff
@@ -208,16 +205,14 @@ def test_main(self, mock_delete, mock_process, mock_find):
     @patch('scripts.manual_operations.manual_remove.ManualRemove.find_runs_in_database')
     @patch('scripts.manual_operations.manual_remove.ManualRemove.process_results')
     @patch('scripts.manual_operations.manual_remove.ManualRemove.delete_records')
-    @patch('scripts.manual_operations.manual_remove.handle_input')
-    def test_main_range(self, mock_handle_input, mock_delete, mock_process, mock_find):
+    @patch('scripts.manual_operations.manual_remove.user_input_check')
+    def test_main_range(self, mock_delete, mock_process, mock_find, mock_uic):
```

Comment:
Could we have 2 more tests here? 
* Test the removal of an individual run and also a range of runs less than 10.
* Test invalid input - so maybe something to asset that an error is raised if the first run > end run